package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "publish",
```

Comment:
Short discussion on naming in the channel. https://botbot.me/freenode/fluid-work/2015-10-20/?msg=52340601&page=1